hack/build/docker.bzl

```diff
@@ -17,6 +17,9 @@ load("@io_bazel_rules_docker//container:bundle.bzl", "container_bundle")
 load("@io_bazel_rules_docker//go:image.bzl", "go_image")
 load("@io_bazel_rules_go//go:def.bzl", "go_test")
 
+
+# TODO: It seems like this code is never used, see https://github.com/jetstack/cert-manager/issues/3072
+# It should be either removed or the work on it finished.
```

Comment:
I'd love to be able to run e2e tests with coverage enabled. I hope we do finish this some day. Thanks for adding the TODO.